DataPlotly/gui/plot_settings_widget.py

```diff
@@ -771,6 +771,10 @@ def refreshWidgets(self):  # pylint: disable=too-many-statements,too-many-branch
             self.y_axis_mode_combo: ['scatter', 'box'],
             self.invert_x_check: ['scatter', 'bar', 'box', 'histogram', '2dhistogram'],
             self.invert_y_check: ['scatter', 'bar', 'box', 'histogram', '2dhistogram'],
+            self.x_axis_min: ['scatter', 'bar', 'box', 'histogram', '2dhistogram', 'ternary', 'violin'],
+            self.x_axis_max: ['scatter', 'bar', 'box', 'histogram', '2dhistogram', 'ternary', 'violin'],
+            self.y_axis_min: ['scatter', 'bar', 'box', '2dhistogram', 'ternary', 'violin'],
+            self.y_axis_max: ['scatter', 'bar', 'box', '2dhistogram', 'ternary', 'violin'],
```

Comment:
I'd avoid this for ternary, I don't thin it is working (values are automatically scaled by plotly) and even if, we should have also a third axis.

Reply:
I have removed ternary